db/20230405185602_initial-schema.down.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS outstanding_batches CASCADE;
```

Comment:
`sqlx migrate add` generates these filenames, and the other `sqlx migrate` subcommands determine the order of migrations by [sorting on the first component of the migration name](https://github.com/launchbadge/sqlx/blob/main/sqlx-core/src/migrate/source.rs). However it doesn't _have_ to be a timestamp. I think any integer would work. But it would be our responsibility to rename the generated migration files from `YYYYMMDDHHMMSS_name.direction.sql` to `00001_name.direction.sql` (or whatever number is appropriate). On balance, I prefer having fewer manual steps when creating migrations. I also don't think using an integer would help with conflicts when two PRs are concurrently adding migrations: if `main` currently went up to migration number 12, and we both made PRs adding different migrations, we'd both choose 13 and then one of us would lose and have to go to 14, which is basically the same thing that would happen if our timestamps collided.